types/selenium-webdriver/remote.d.ts

```diff
@@ -76,7 +76,6 @@ export namespace DriverService {
      *
      * @param {...CommandLineFlag} var_args The arguments to include.
      * @return {!THIS} A self reference.
```

Comment:
these ought to be deleted too

Reply:
I ended up reverting this back to `master`. +1 to fixing it up, but I'm running low on time budget...